pandas-stubs/core/generic.pyi

```diff
@@ -163,15 +166,17 @@ class NDFrame(PandasObject, indexing.IndexingMixin):
     def to_sql(
         self,
         name: _str,
-        con,
+        con: Any,
         schema: _str | None = ...,
         if_exists: _str = ...,
         index: _bool = ...,
-        index_label: _str | Sequence[_str] | None = ...,
+        index_label: IndexLabel = ...,
         chunksize: int | None = ...,
-        dtype: dict | Scalar | None = ...,
-        method: _str | Callable | None = ...,
-    ) -> None: ...
+        dtype: DtypeArg | None = ...,
+        method: Literal["multi"]
+        | None
+        | Callable[[SQLTable, Any, list[str], Iterable], int | None] = ...,
```

Comment:
We've been using the convention to put `None` last so can you exchange the order of `Callable` and `None` here?

Reply:
Fixed.